EduVPN-macOS/Controllers/ProvidersViewController.swift

```diff
@@ -32,6 +32,11 @@ class ProvidersViewController: NSViewController {
     var selectingConfig: Bool = false
     
     var providerType: ProviderType = .unknown
+
+    var configuredForInstancesDisplay: Bool {
```

Comment:
This seems to be named for the opposite? Wouldn’t you be unconfigured when provider type is unknown? Or I am misunderstanding what you mean with “instances display”.

Reply:
Well, just the fact we are discussing this means the naming can be better. What I intend to communicate is that if this boolean is true, then the VC is configured to display a list of instances.